packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFDataConverters.m

```diff
@@ -167,7 +167,7 @@ WKAudiovisualMediaTypes FWFNativeWKAudiovisualMediaTypeFromEnumData(
 
 FWFNSUrlRequestData *FWFNSUrlRequestDataFromNativeNSURLRequest(NSURLRequest *request) {
   return [FWFNSUrlRequestData
-              makeWithUrl:request.URL.absoluteString
+              makeWithUrl:request.URL.absoluteString.length > 0 ? request.URL.absoluteString : @""
```

Comment:
im a bit confused here - will absoluteString ever be `nil`? if so, may be less confusing to do nil check rather than length check?

Reply:
https://github.com/flutter/packages/blob/83c2c4da97c554996c488037c7ca7c1d5a076a9b/packages/webview_flutter/webview_flutter_wkwebview/example/integration_test/legacy/webview_flutter_test.dart#L1124-L1149

@hellohuanlin 
It can be nil with this test case.
I updated code to nil check.
Thanks